R/plot-build.r

```diff
@@ -44,6 +44,8 @@ ggplot_build.ggplot <- function(plot) {
     out
   }
 
+  plot$theme <- plot_theme(plot)
```

Comment:
This should be commented, just like the other parts of the code in this function.